LibreHardwareMonitorLib/Hardware/IHardware.cs

```diff
@@ -21,7 +21,8 @@ public enum HardwareType
         Heatmaster,
         Storage,
         Network,
-        AeroCool
+        AeroCool,
+        LiquidCooler
```

Comment:
Either this should be named as Nzxt or the other water cooling stuff should be moved into this.

Reply:
I was contemplating both moves, which way would you like to see this go?
It would make sense to make a hardware type category by device type.
Currently there are manufacturer specific implementations with special icons for them in the GUI, is it okay if we lose those or find a different way to determine what manufacturers they are?

Reply:
PS: Reason I'm asking, I also have access to a Corsair Hydro H115i, which I partially got working already (its just not in my primary system, so I'm spending less time on it than the NZXT).

Reply:
I prefer to move the other ones in one category, as this is similar to (most of) the rest. One liquid cooling icon is OK.

Reply:
I have moved all the cooling items into a new category called "Coolers" because some of these manufacturer specific types can be used as fan controllers only.